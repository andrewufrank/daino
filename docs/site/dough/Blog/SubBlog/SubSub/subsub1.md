---
title: postwk.md
author: AUF
date: Jan. 4, 2019
keywords: test
abstract: A silly text not needing an abstract.
pageTemplate: page3.yaml
---

Some text without meaning;  postwk.



  An example post.  
