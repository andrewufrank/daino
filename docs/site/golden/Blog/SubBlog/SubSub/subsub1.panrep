Panrep {panyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/SubSub/subsub1.md", dyLink = "Blog/SubBlog/SubSub/subsub1.md", dyLang = DLenglish, dyTitle = "postwk.md", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/SubSub/subsub1.md", link = "Blog/SubBlog/SubSub/subsub1.md", title = "postwk.md", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, panpan = Pandoc (Meta {unMeta = fromList [("abstract",MetaInlines [Str "A",Space,Str "silly",Space,Str "text",Space,Str "not",Space,Str "needing",Space,Str "an",Space,Str "abstract."]),("author",MetaInlines [Str "AUF"]),("date",MetaInlines [Str "Jan.",Space,Str "4,",Space,Str "2019"]),("keywords",MetaInlines [Str "test"]),("pageTemplate",MetaInlines [Str "page3.yaml"]),("title",MetaInlines [Str "postwk.md"])]}) [Para [Str "the",Space,Str "title",Space,Str "should",Space,Str "become"],RawBlock (Format "tex") "\\chapter{postwk.md}",Para [Str "Some",Space,Str "text",Space,Str "without",Space,Str "meaning;",Space,Str "postwk."],Header 2 ("a-subsection-i-assume",[],[]) [Str "a",Space,Str "subsection",Space,Str "i",Space,Str "assume"],Para [Str "with",Space,Str "some",Space,Str "text",SoftBreak,Str "and",Space,Str "a"],Header 3 ("subsubsection",[],[]) [Str "subsubsection."],Para [Str "An",Space,Str "example",Space,Str "post."]]}