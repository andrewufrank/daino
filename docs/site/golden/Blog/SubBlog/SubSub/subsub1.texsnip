TexSnip {snipyam = MetaPage {dyFn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/SubSub/subsub1.md", dyLink = "Blog/SubBlog/SubSub/subsub1.md", dyLang = DLenglish, dyTitle = "postwk.md", dyAbstract = "A silly text not needing an abstract.", dyAuthor = "AUF", dyDate = Just "Jan. 4, 2019", dyKeywords = "test", dyBibliography = Nothing, dyStyle = Nothing, dyReferences = Nothing, dyNoCite = Nothing, dyPublish = Nothing, dyIndexPage = False, dyIndexSort = Nothing, dyIndexEntry = IndexEntry {fn = "/home/frank/Workspace11/ssg/docs/site/dough/Blog/SubBlog/SubSub/subsub1.md", link = "Blog/SubBlog/SubSub/subsub1.md", title = "postwk.md", abstract = "A silly text not needing an abstract.", author = "AUF", date = "Jan. 4, 2019", publish = Nothing, indexPage = False, dirEntries = [], fileEntries = []}}, unTexSnip = "the title should become\n\n\\chapter{postwk.md}\n\nSome text without meaning; postwk.\n\n\\hypertarget{a-subsection-i-assume}{%\n\\subsection{a subsection i assume}\\label{a-subsection-i-assume}}\n\nwith some text and a\n\n\\hypertarget{subsubsection}{%\n\\subsubsection{subsubsection.}\\label{subsubsection}}\n\nAn example post."}