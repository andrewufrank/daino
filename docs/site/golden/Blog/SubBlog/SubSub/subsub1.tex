\documentclass[a4paper,10pt]{scrbook}
\usepackage{fontspec}
\usepackage[ngerman]{babel}
\usepackage{graphicx}
\usepackage{makeidx}
\usepackage{natbib}
\makeindex
\usepackage[colorlinks]{hyperref}
\providecommand{\tightlist}{%
\setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}

\begin{document}

the title should become

\chapter{postwk.md}

Some text without meaning; postwk.

\hypertarget{a-subsection-i-assume}{%
\subsection{a subsection i assume}\label{a-subsection-i-assume}}

with some text and a

\hypertarget{subsubsection}{%
\subsubsection{subsubsection.}\label{subsubsection}}

An example post.


\printindex
\end{document}
